lib/fastly.rb

```diff
@@ -10,7 +10,8 @@ class Fastly
   # These are not kwargs because delayed_job doesn't correctly support kwargs in Fastly.delay.purge
   # See: https://github.com/collectiveidea/delayed_job/issues/1134
   def self.purge(options = {})
-    return unless ENV["FASTLY_DOMAINS"]
+    return unless ENV["FASTLY_DOMAINS"] && ENV["FASTLY_API_KEY"]
```

Comment:
Tiny nit-pick, but maybe also add a `present?` check to ensure the environment variables are both present and not empty strings?

Reply:
let's do that :muscle:

Reply:
updated